### PR TITLE
Remove document.xml_importer

### LINKED ISSUE #923: Remove document.xml_importer
With #868 and #884 landing, I *think* we can safely remove `document.xml_importer`, if we want, by:

1. loading the AKN representation for our two current XML docs in our AKN editor,
2. replacing their respective XML files in the repo with the content from step 1,
3. modifying our `import_xml_doc` command to use our AKN XML parser to bring the document into the database, rather than `document.xml_importer`.

### PR DESCRIPTION
**Pull request by toolness**
This fixes #923 by:

1. loading the AKN representation for our two current XML docs in our AKN editor,
2. replacing their respective XML files in the repo with the content from step 1,
3. modifying our `import_xml_doc` command to use our AKN XML parser to bring the document into the database, rather than `document.xml_importer`.
4. Removing `document.xml_importer`.

### CODE REVIEW

File: api/document/management/commands/import_xml_doc.py [hunk 1]
```diff
@@ -17,6 +18,14 @@ def fetch_policy(identifier: str):
         return Policy.objects.filter(omb_policy_id=identifier).first()
 
 
+def import_xml_doc(policy, xmlstream):
+    parsed_data = AkomaNtosoParser().parse(xmlstream)
+    cursor = DocCursor.new_tree('', policy=policy)
```
Comment (toolness) [inline api/document/management/commands/import_xml_doc.py]: Aggh I _really_ dislike this line.  Our `DocCursorSerializer` was built to serialize existing `DocCursor`s, not to create brand-new ones. So here I'm just "creating" a new `DocCursor` for no other reason than so our importer can eventually pluck out its `policy`. Ooooof.

File: api/document/tests/import_xml_doc_test.py [hunk 2]
```diff
@@ -15,3 +19,30 @@ def test_fetch_policy_pk():
 def test_fetch_policy_number():
     policy = mommy.make(Policy, omb_policy_id='M-12-13')
     assert import_xml_doc.fetch_policy('M-12-13') == policy
+
+
+@pytest.mark.django_db
+def test_import_xml_doc():
```
Comment (toolness) [inline api/document/tests/import_xml_doc_test.py]: This is really just an old test from `xml_importer`, but I made it run twice just to make sure the code is OK with both creating a new `DocNode` tree where none existed before, _and_ with overwriting an existing `DocNode` tree if it already exists.

File: api/document/management/commands/import_xml_doc.py (hunk 1 above)
Comment (cmc333333) [reply api/document/management/commands/import_xml_doc.py]: Would it make sense to override the serializer's `create()` method? We'd need to pass the policy as context, but I wonder if that's where folks would expect this sort of logic?

File: api/document/management/commands/import_xml_doc.py [hunk 3]
```diff
@@ -17,6 +18,14 @@ def fetch_policy(identifier: str):
...
+    serializer = DocCursorSerializer(cursor, data=parsed_data)
+    serializer.is_valid(raise_exception=True)
```
Comment (cmc333333) [inline api/document/management/commands/import_xml_doc.py]: I'm betting this raises some gnarly, but data-rich exceptions. Not necessary for this round, but we should think about ways to surface them to users.

File: api/document/management/commands/import_xml_doc.py (hunk 1 above)
Comment (toolness) [reply api/document/management/commands/import_xml_doc.py]: Ah that's a great idea! I just filed #950 to address it.

**Pull request merged**